package.json

```diff
@@ -41,6 +42,7 @@
     "web3": "1.3.4"
   },
   "peerDependencies": {
-    "puppeteer": ">5"
+    "puppeteer": ">7",
```

Comment:
```suggestion
    "puppeteer": ">5",
```

Reply:
it is updated to version 7, not sure it will work with a lower version of a puppeteer